fixieai/agents/code_shot.py

```diff
@@ -30,6 +30,13 @@
 _AGENT_ID_JWT_CLAIM = "aid"
 
 
+@pydantic_dataclasses.dataclass
+class ModelSettings:
```

Comment:
please add docstring for this class and the fields knowing that it will get exported as docs